### PR TITLE
Initial project OWNERS and GOVERNANCE under the LinuxFoundation

### PR DESCRIPTION
**Pull request by armab**
Per previous discussions and drafts, establish initial/seed project Maintainers and structure for StackStorm Technical Steering Committee under the LinuxFoundation.
Define different groups, including Maintainers/Contributors, their responsibilities and expectations.
Document processes around voting, proposing new maintainers and more.

Technical Steering Committee:
- [x] @dzimine
- [x] @punkrokk
- [x] @LindsayHill
- [x] @armab
- [x] @nmaludy
- [x] @blag
- [x] @bigmstone
- [x] @kami
- [x] @mickmcgrath13
- [x] @m4dcoder

----

Special :heart: thanks to those who participated in the past and formed the project we have today. 

@StackStorm/alumni and @nzlosh @userlocalhost @jjm @shusugmt @trstruth @tonybaloney @AndyMoore @humblearner @dennybaa @emedvedev  @epowell101  @enykeev @mward29 @Mierdin  @DoriftoShoes @jfryman @jinpingh  @johandahlberg @johanherman @lakshmi-kannan @manasdk @VineeshJain @warrenvw.

### CODE REVIEW

**Comment by punkrokk** [comment]
Any interest in a PR template for adding people and linking the file to .github/OWNERS file? I can push a commit showing what I mean later today

Cheers,

JP

**Comment by armab** [comment]
@punkrokk I think you meant `CODEOWNERS` https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
     + :100: Please feel free to push the change.

**Comment by punkrokk** [comment]
@armab I did, and pushed.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+StackStorm has joined LinuxFoundation in 2019.
+With the new Neutral Home we invite everyone: organizations, teams and individuals relying on StackStorm Automation in their operations to support the project by giving back and participating in development as well as influence on its future under the Open Governance.
+
+# StackStorm Governance
+This document defines governance policies for the StackStorm project.
+It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2019/10/StackStorm-Technical-Charter-20191004.pdf) for StackStorm within LinuxFoundation.
```
Comment (nmaludy) [inline GOVERNANCE.md]: LinuxFoundation should be two words

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  * Receive **three votes** in the [conflict resolution and voting process](#conflict-resolution-and-voting) described below.
+* [Senior Maintainers](OWNERS.md#senior-maintainers-)
+  * Have the most in-depth experience with the StackStorm project and are expected to have the knowledge and insight to lead the project's future, growth, standards and improvement.
+  * Oversee the process for adding new maintainers and provide guidance, help and sharing their experience with the standard maintainers.
+  * Have full owner access to all the resources and platforms that sustain StackStorm project.
+  * Receive **two votes** in the voting process.
+* [Maintainers](OWNERS.md#maintainers-)
+  * Have good experience with the StackStorm codebase, expected to provide significant value to the project, helping it grow, improve and succeed.
+  * Have full member write access to [StackStorm](https://github.com/stackstorm/) and [StackStorm-Exchange](https://github.com/stackstorm-exchange) Github organizations, Moderator at [forum](https://forum.stackstorm.com/), [Slack](https://stackstorm.com/community-signup) and other Community platforms.
```
Comment (nmaludy) [inline GOVERNANCE.md]: Should maintainers also have access to CI so they can force rebuild if necessary?

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  * see all the Github [**labels**](https://github.com/stackstorm/st2/labels). Recommended informative balance for the issue is 3-5 labels.
+* Make sure that ongoing PRs are moving forward at the right pace or closing them. It's important to apply the same to old PRs and Issues to retain healthy project state.
+* Guide Community about using the right channel: Github for Issues and Bug-reports, while Forums and Slack for questions and discussions.
+* Project maintenance: security, updates, CI/CD, builds, infrastructure.
+* Prioritize the work following [StackStorm Roadmap](https://docs.stackstorm.com/latest/roadmap.html) to move the project forward.
+* All maintainers ideally are expected to shoulder a proportional share of community work: Issues, Reviews, Slack, Forum, Releases, fixing CI/CD Builds, etc. Work upon to make sure the balance is kept within the team.
+* Encourage and guide other community members to contribute back to StackStorm. Even a simple bug report or a one-line documentation PR is a win!
+* Follow-up by filling undocumented issues found in community channels, show an example.
```
Comment (nmaludy) [inline GOVERNANCE.md]: Filing not "filling"

File: OWNERS.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+Have deep platform knowledge & experience and demonstrate technical leadership as well as driving the project forward.
+* Andrew Hubl ([@blag](https://github.com/blag)) <<dhubl@stackstorm.com>>
+  - ChatOps, StackStorm Exchange, Community, Documentation, CI/CD.
+* Eugen Cusmaunsa ([@armab](https://github.com/armab)) <<armab@stackstorm.com>>
+  - Systems, Deployments, Docker, K8s, HA, Ansible, Chef, Vagrant, deb/rpm, CI/CD, Infrastructure, Release Engineering, Community.
+* Matthew Stone ([@bigmstone](https://github.com/bigmstone)) <<matt@stackstorm.com>>
+  - Core, External Relations, Robot Arm [:)](https://twitter.com/Stack_Storm/status/1217056819736203270)
+* Tomaz Muraus ([@kami](https://github.com/kami)) <<tomaz@stackstorm.com>>
+  - Core, Performance, API, Scalability, CI/CD, Systems, Deployments, Packaging, OpenSource.
+* Winson Chan ([@m4dcoder](https://github.com/m4dcoder)) <<wcchan@stackstorm.com>>
```
Comment (trstruth) [inline OWNERS.md]: Winston?

Comment (armab) [reply OWNERS.md]: Winson

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# Set TSC Voting Members as PR Reviewers for Changes to GOVERNANCE.md and OWNERS documents
+GOVERNANCE.md @StackStorm/Maintainers @StackStorm/Leaders
```
[the remark this answers is not in the document]
Comment (armab) [reply .github/CODEOWNERS]: Senior maintainers are part of https://github.com/orgs/StackStorm/teams/maintainers, with no special Github team for simplicity reasons.

See how OWNERS.md highlights them:
> Senior [@StackStorm/maintainers](https://github.com/orgs/StackStorm/teams/maintainers)

Also technically, @StackStorm/maintainers and @StackStorm/leaders are subgroups under @StackStorm/tsc so its possible to use tsc shortcut to mention everyone.

File: GOVERNANCE.md (hunk 1 above)
Comment (armab) [reply GOVERNANCE.md]: probably even 3

File: GOVERNANCE.md (hunk 3 above)
Comment (armab) [reply GOVERNANCE.md]: Thanks, fixed

File: GOVERNANCE.md (hunk 2 above)
Comment (armab) [reply GOVERNANCE.md]: :+1: Added CI/CD to be explicit.

File: OWNERS.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-StackStorm was founded in 2013. After several acquisitions project has finally joined [LinuxFoundation](https://www.linuxfoundation.org/projects/directory/) in 2019.<br>
+StackStorm was founded in 2013. After several acquisitions the project has finally joined [The Linux Foundation](https://www.linuxfoundation.org/projects/directory/) in 2019.<br>
```
Comment (armab) [inline OWNERS.md]: @LindsayHill  Thanks and welcome back! :heart:

File: .github/CODEOWNERS (hunk 5 above)
[the remark this answers is not in the document]
Comment (punkrokk) [reply .github/CODEOWNERS]: @nmaludy also senior maintainers are in the maintainers group

[the remark this answers is not in the document]
Comment (armab) [reply .github/CODEOWNERS]: It's also a good question. This information might be useful for others.

**Comment by blag** [comment]
Sorry for the rebase and force push, I wanted to remove my real name and contact info from the PR.

**Comment by dzimine** [comment]
LGTM and thanks for doing the heavy-lifting! 👍

**Pull request merged**